src/components/ShareImage/ShareImagePreview.js

```diff
@@ -167,6 +167,7 @@ const ShareImagePreview = ({
             fontSize: fontSize || SHARE_IMAGE_DEFAULTS.fontSize,
             color: inverted ? '#FFF' : '#000',
             width: shareImage && '80%',
+            lineHeight: 1.25,
```

Comment:
The current default value in Chrome seems to be 1.15. Could this increase potentially lead to old image being to big?

Reply:
yes. is there a way around? It should have been 1.25 from the beginning. We could limit potential problems by limiting the line height changes to formats that have gt-america, but it would mean that other formats would remain with narrow line-heights...

Reply:
since it hasn't been that long since we introduced the feature, I'd volunteer to go back to all 30 or so articles to check and clean up errros.

Reply:
which we need to do anyways, because there might be some issues with the lowercase characters "g" "j" etc.